### PR TITLE
Initial commit for Trellis M4 Express library

### PR DESCRIPTION
**Pull request by kattni**
Added `pixels.brightness`, `pixels.width`, `pixels.height`, and `pixels.fill`. 

Included a simpletest demo that starts out rainbow-lit and then lights up all LEDs the same color as the button pressed as a way to test whether your buttons work without using the serial output.

### CODE REVIEW

File: adafruit_trellism4.py [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+                import adafruit_trellism4
+
+                trellis = adafruit_trellism4.TrellisM4Express()
+
+                trellis.pixels.brightness = 0.3
+
+                trellis.pixels.fill((255, 0, 0))
+        """
+
+        print(dir(board))
```
Comment (sommersoft) [inline adafruit_trellism4.py]: I leave these all over the place too! 😄

File: docs/index.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    api
+
+.. toctree::
+    :caption: Tutorials
+
+
+.. toctree::
+    :caption: Related Products
+
```
Comment (sommersoft) [inline docs/index.rst:29]: Copy + Paste:
`Adafruit NeoTrellis RGB Driver PCB for 4x4 Keypad <https://www.adafruit.com/product/3954>`

**Review by sommersoft — CHANGES_REQUESTED** [review]
Looks good to me! Couple small things...

File: docs/index.rst (hunk 2 above)
Comment (sommersoft) [reply docs/index.rst:29]: conflation...and hunger. never a good mix.

Comment (kattni) [reply docs/index.rst:29]: I don't think it works with the NeoTrellis. Good thought though!

**Review by sommersoft — APPROVED** [review]

**Review by ladyada — APPROVED** [review]

**Pull request merged**